src/me/mrCookieSlime/Slimefun/api/TickerTask.java

```diff
@@ -202,20 +205,27 @@ public void info(CommandSender sender) {
 			tellraw.addText("   &7&oHover for more Info");
 			StringBuilder hover = new StringBuilder();
 			int hidden = 0;
-			
-			for (Map.Entry<String, Integer> entry: machineCount.entrySet()) {
-				long timings = machineTimings.get(entry.getKey());
-				if (timings > 0) hover.append("\n&c").append(entry.getKey()).append(" - ").append(entry.getValue()).append("x &7(").append(timings).append("ms)");
-				else hidden++;
+
+			Map<String, Long> timings = machineCount.entrySet().stream()
+					.sorted(Comparator.comparingLong(machineTimings::get).reversed())
```

Comment:
This is a Stream of Map.Entry<String, Long>
machineTimings::get will always return null as that map's key is not an entry...
You need to pass the entry's key instead of just giving it the function there.